packages/react-ai/src/components/AIConversation/AIConversation.tsx

```diff
@@ -41,11 +41,11 @@ function AIConversationBase({
   const defaultAvatars: Avatars = {
     ai: {
       username: 'Assistant',
-      avatar: icons?.assistant ?? <IconAssistant />,
+      avatar: icons?.assistant ?? <IconAssistant testId="icon-assistant" />,
```

Comment:
What's the reasoning behind the `testId` values?

Reply:
So I can easily check for them in unit tests. Should I not add testIds like this?

Reply:
Been trying to use roles for element lookups more recently since our test id approach is inconsistent, also somewhat lean towards the thtought we shouldn't hard code them on public components to avoid conflicts with consumer test ids (however unlikely that may be)